packages/apps/crm/src/screens/opportunity/OpportunityFormScreen.js

```diff
@@ -118,55 +127,90 @@ const OpportunityFormScreen = ({navigation, route}) => {
             styleTxt={styles.marginTitle}
+            required={true}
           />
           <DateInput
             title={I18n.t('Crm_Opportunity_ExpectedCloseDate')}
-            defaultDate={date}
-            onDateChange={setDate}
+            defaultDate={
+              idOpportunity != null
+                ? new Date(_opportunity.expectedCloseDate)
+                : null
+            }
+            onDateChange={value =>
+              handleOpportunityFieldChange(
+                value?.toISOString().split('T')[0],
+                'expectedCloseDate',
+              )
+            }
             style={styles.input}
           />
           <FormIncrementInput
             title={I18n.t('Crm_Opportunity_Amount')}
-            defaultValue={amount}
-            onChange={setAmount}
+            defaultValue={idOpportunity != null ? _opportunity.amount : 0}
+            onChange={value => handleOpportunityFieldChange(value, 'amount')}
             decimalSpacer={I18n.t('Base_DecimalSpacer')}
             thousandSpacer={I18n.t('Base_ThousandSpacer')}
           />
           {crmConfig?.isManageRecurrent && (
             <FormIncrementInput
               title={I18n.t('Crm_Opportunity_RecurrentAmount')}
-              defaultValue={recurrent}
-              onChange={setRecurrent}
+              defaultValue={
+                idOpportunity != null ? _opportunity.recurrentAmount : 0
+              }
+              onChange={value =>
+                handleOpportunityFieldChange(value, 'recurrentAmount')
+              }
               decimalSpacer={I18n.t('Base_DecimalSpacer')}
               thousandSpacer={I18n.t('Base_ThousandSpacer')}
             />
           )}
           <FormHtmlInput
             title={I18n.t('Base_Description')}
-            onChange={setDescription}
-            defaultValue={description}
+            onChange={value =>
+              handleOpportunityFieldChange(value, 'description')
+            }
+            defaultValue={_opportunity.description}
           />
           <Picker
             style={[styles.picker, styles.marginPicker]}
             styleTxt={styles.marginPicker}
             title={I18n.t('Crm_Opportunity_Status')}
-            defaultValue={status}
+            defaultValue={_opportunity.opportunityStatus?.id}
             listItems={opportunityStatusList}
             labelField="name"
             valueField="id"
             emptyValue={false}
-            onValueChange={setStatus}
+            onValueChange={value =>
+              handleOpportunityFieldChange(value, 'opportunityStatus')
+            }
             isScrollViewContainer={true}
+            required={true}
           />
         </View>
       </KeyboardAvoidingScrollView>
-      <View style={styles.button_container}>
-        <Button title={I18n.t('Base_Save')} onPress={updateOpportunityAPI} />
-      </View>
+      <ValidateButtonOpportunity
+        _opportunity={_opportunity}
+        company={company}
```

Comment:
The company is only used there, maybe you can call `useSelector` directly inside `ValidateButtonOpportunity` to avoid useless attribut on this screen